lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -116,16 +116,15 @@ abstract class SqlParserTestBase : TestBase() {
     private fun unwrapQuery(statement: PartiqlAst.Statement) : SexpElement {
        return when (statement) {
            is PartiqlAst.Statement.Query -> statement.expr.toIonElement()
-           is PartiqlAst.Statement.Dml -> statement.toIonElement()
+           is PartiqlAst.Statement.Dml,
```

Comment:
For the comment https://github.com/partiql/partiql-lang-kotlin/pull/462#discussion_r739469077